source/manual/gtm-change-process.html.md

```diff
@@ -0,0 +1,62 @@
+---
+owner_slack: "#govuk-frontenders"
+title: 'Google Tag Manager change process'
+section: Frontend
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+related_repos: [govuk_publishing_components]
+---
+
+GOV.UK has implemented Google Analytics 4 (GA4) using Google Tag Manager (GTM) to track user journeys through the site. This page describes the process of making changes to GTM to support improvements to our analytics.
+
+This process exists because some changes can increase the size of the JavaScript included from GTM in the public frontend of GOV.UK. We do not want to unnecessarily degrade the performance of GOV.UK.
+
+## Roles and responsibilities
+
+Performance analysts:
+
+- to make changes in GTM and share the change with a succinct explanation of what it does and why it's needed
+- to fully document the details of the change
+
+Developers:
+
+- to review and publish the change in GTM
+- to confirm that the changes won't break anything
```

Comment:
Just thinking, if we go the normal route: Integration -> Staging -> Production, is this step is necessary. Wouldn't any breaking change be caught in Integration? I guess it's worth keeping in, but I wouldn't expect anything to break in Staging or Production if it's ok in integration.